aacgmv2/__init__.py

```diff
@@ -41,9 +45,16 @@
 """
 import os as _os
 from sys import stderr
-import logbook as logging
+import logging
 
-__version__ = "2.5.1"
+__version__ = "2.5.2"
+
+# Define a logger object to allow easier log handling
+logger = logging.getLogger('aacgmv2_logger')
+
+# Altitude constraints
+high_alt_coeff = 2000.0 # Tested and published in Shepherd (2014)
+high_alt_trace = 6378.0 # 1 RE, these are ionospheric coordinates
 
 # path and filename prefix for the IGRF coefficients
 AACGM_v2_DAT_PREFIX = _os.path.join(_os.path.realpath( \

```

Comment:
This is something that RST and older AACGM users probably have in their .bashrc/.cshrc files, and could cause issues.  So I made a workaround that allows people (through a specific option) to use the environment variables, but ignores them by default.  It prints out a warning that the coefficients included in this install are being used instead of the ones they have declared locally so that users can remember to change things if they want to.  If you don't have these defined, the module defines them without any warning messages.